corehq/motech/openmrs/tasks.py

```diff
@@ -144,7 +149,7 @@ def import_patients_of_owner(requests, importer, domain_name, owner, location=No
     submit_case_blocks(
         [cb.case.as_string() for cb in case_blocks],
         domain_name,
-        username=owner.username,
+        device_id=OPENMRS_IMPORTER_DEVICE_ID + importer.get_id,
```

Comment:
nit: I think `'{}{}'.format(OPENMRS_IMPORTER_DEVICE_ID, importer.get_id)` would be a slightly more safe way to do this (though doesn't matter if both are always strings).

Reply:
:+1: It's a good principle regardless.